### PR TITLE
test: add `pox-reject` test

### PR DESCRIPTION
**Pull request by obycode**

### CODE REVIEW

File: tests/integration/pox/pox-3/reject-pox.spec.ts [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    poxInfo = await getPoxInfo(network);
+    expect(poxInfo.rejection_votes_left_required).toBe(0);
+
+    isPoxActive = await callReadOnlyIsPoxActive(
+      3,
+      network,
+      Accounts.WALLET_1,
+      poxInfo.current_cycle.id
+    );
+    expect(isPoxActive).toEqual(trueCV());
```
Comment (friedger) [inline tests/integration/pox/pox-3/reject-pox.spec.ts:183]: Why true here? It was false in the previous cycle for the next cycle

File: tests/integration/pox/pox-3/reject-pox.spec.ts [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+    let poxInfo = await getPoxInfo(network);
+    expect(poxInfo.rejection_votes_left_required).toBeGreaterThan(0);
+
+    let isPoxActive = await callReadOnlyIsPoxActive(
+      3,
+      network,
+      Accounts.WALLET_1,
+      poxInfo.current_cycle.id
+    );
+    expect(isPoxActive).toEqual(falseCV());
```
Comment (friedger) [inline tests/integration/pox/pox-3/reject-pox.spec.ts:200]: Why false here? Or is it enabled in the next cycle?

**Review by friedger — COMMENTED** [review]
Not sure what the rules for pox-reject are. It is not clear to me from the test.

File: tests/integration/pox/pox-3/reject-pox.spec.ts (hunk 1 above)
Comment (obycode) [reply tests/integration/pox/pox-3/reject-pox.spec.ts:183]: Yeah, you're right. I think I made a mistake in the test case here and it accidentally worked. I'll look into it.

Comment (obycode) [reply tests/integration/pox/pox-3/reject-pox.spec.ts:183]: Ok, I need to go one block more to get the data for the right cycle. I'll push an update soon. Thanks for catching that.

Comment (obycode) [reply tests/integration/pox/pox-3/reject-pox.spec.ts]: Fixed in c32a496.

File: tests/integration/pox/pox-3/reject-pox.spec.ts (hunk 2 above)
Comment (obycode) [reply tests/integration/pox/pox-3/reject-pox.spec.ts]: See above. Fixed in c32a496

**Pull request merged**